packages/core/tests/__snapshots__/config.test.ts.snap

```diff
@@ -7,6 +7,7 @@ exports[`Should compose create Rsbuild config correctly > Merge Rsbuild config 1
       "dev": {
         "progressBar": false,
       },
+      "format": "esm",
```

Comment:
We're adding unknown field to Rsbuild config here, could we prevent this?

Reply:
Just add format this time, and other unknown field is already there for severral days. Rsbuild inspect seems to filter them in display.

Reply:
> and other unknown field is already there for severral days

Did we? The seems not in the snapshot at the moment.

Reply:
because we only add format in our unit test